config/config_test.go

```diff
@@ -332,7 +333,18 @@ func TestLoadBoilerPlateConfigFullConfig(t *testing.T) {
 func TestLoadBoilerPlateConfigNoConfig(t *testing.T) {
 	t.Parallel()
 
-	actual, err := LoadBoilerPlateConfig(&BoilerplateOptions{TemplateFolder: "../test-fixtures/config-test/no-config"})
+	templateFolder := "../test-fixtures/config-test/no-config"
+	_, err := LoadBoilerPlateConfig(&BoilerplateOptions{TemplateFolder: templateFolder})
```

Comment:
Good point. I had a few instances of this. All fixed here: https://github.com/gruntwork-io/boilerplate/commit/94781854c7205d23f41c736bd9c4d704ed112763